src/main/g8/src/test/resources/$package$/$name__camel$/Test$name__Camel$.tdml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+$!
```

Comment:
why do so many of these files have "_Camel" in their template names?

Reply:
It's g8 syntax to format the name property. name__Camel means take the "name" property and make it camel case with the first letter capitalized. name__camel means take the "name" property and make it camel case with the first letter lowercase.

So if you generate a project with name "My Great Format", in creates myGreatFormat/TestMyGreatFormat.tdml

There's lots of other formatting capabilities: http://www.foundweekends.org/giter8/formatting.html

Note that the __ stuff is used only for directory/file names, since windows doesn't like the ``;format="..."`` syntax that used inside files.

Reply:
Ah,  I thought it was some holdover from using this tool for Apache Camel or something. This makes perfect sense. Pretty clever actually.